package.json

```diff
@@ -50,7 +50,7 @@
     "appium-ios-simulator": "^3.28.0",
     "appium-remote-debugger": "^8.13.2",
     "appium-support": "^2.47.1",
-    "appium-webdriveragent": "^3.8.0",
+    "appium-webdriveragent": "^3.9.1",
```

Comment:
Are you sure it is 3.9.1? I saw the latest is [3.8.1](https://github.com/appium/WebDriverAgent/releases/tag/v3.8.1) though

Reply:
Yes. I pushed 3.9.0 before but the version in the package.json was back to 3.8.0 accidentally. Then, I pushed a patch as 3.8.1. After noticed it, I repushed 3.9.1...